### ISSUE TITLE
OFFSPEC: Locate driver for ISIS Enviornmental Monitor

### ISSUE THREAD

**Issue by aaron-long** [ISISComputingGroup/IBEX#4819]
Instrument scientist called as they were unable to locate the VI driver for the ISIS created Environmental Monitor. After checking we were able to locate the correct driver (`ISIS Environment Monitor`) in the labview drivers directory.

We spent time ensuring the device booted and the sensors connected to the VI successfully.

**Comment by davidkeymer** [comment]
Just to clarify, we were asked to find the VI for an ISIS-built "Humidity" monitor.  It was then deduced that the actual device in use was the "ISIS Environment Monitor", as it monitors humidity and a few other conditions.  Knowing this, the correct VI was selected and configured.

**Comment by kjwoodsISIS** [comment]
I find it hard to believe that this was "0 point" ticket.  You say things like "After checking ..." , "We spent time ensuring ..." and "It was then deduced ...".  These phrases all imply a degree of complexity - you need to check things, work things out, test that things are working correctly.

**Comment by aaron-long** [comment]
@kjwoodsISIS The points I assigned are what I intended. David and I spent 5-10 mins looking for the driver in the labview directory. Bis and I walked down to halls (and within that time the IS had located the driver as we arrived), so we waited for 10 mins to ensure it was working (and I thought it would be nice for Bis to interact and see the use of SE kit software). This was about 30 mins of work total.
